examples/tests.py

```diff
@@ -892,3 +893,43 @@ def test_user_delete(self):
         response = self.client.post('/admin/examples/ldapuser/foouser/delete/',
                                     {'yes': 'post'})
         self.assertRedirects(response, '/admin/examples/ldapuser/')
+
+
+class FooGroupTestCase(BaseTestCase):
+    directory = dict([groups, foogroup, bargroup, wizgroup, people, foouser])
+
+    def as_ldap_query(self, qs):
+        connection = connections['ldap']
+        compiler = SQLCompiler(
+            query=qs.query,
+            connection=connection,
+            using=None,
+        )
+        return query_as_ldap(qs.query, compiler, connection)
+
+    def test_count_all(self):
+        qs = FooGroup.objects.all()
+        lq = self.as_ldap_query(qs)
+        self.assertEqual(lq.base, groups[0])
+        self.assertEqual(lq.filterstr, '(&(objectClass=posixGroup)(cn=foo*))')
+        self.assertEqual(qs.count(), 1)
+
+    def test_base_manager_dn(self):
+        dn = foogroup[0]
+        qs = FooGroup._base_manager.using(None).filter(dn=dn)
```

Comment:
This line mirrors the code inside `ldapdb.models.Model._save_table()`, but swaps in `filter` for `get` so that we can inspect the generated ldap.backends.ldap.compiler.LdapLookup object.